Rock.Messaging.SQS/Configuration/SQSConfiguration.cs

```diff
@@ -1,31 +1,20 @@
 ﻿using System;
-using System.Xml.Serialization;
 
-namespace Rock.Messaging.SQS
+namespace RockLib.Messaging.SQS
 {
-    public class SQSConfiguration : ISQSConfiguration
+    public class SQSConfiguration: ISQSConfiguration
     {
         public SQSConfiguration()
         {
             MaxMessages = 3;
             AutoAcknowledge = true;
         }
 
-        [XmlAttribute("name")]
-        public string Name { get; set; }
-
-        [XmlAttribute("queueUrl")]
-        public string QueueUrl { get; set; }
-
-        [XmlAttribute("maxMessages")]
-        public int MaxMessages { get; set; }
-
-        [XmlAttribute("autoAcknowledge")]
-        public bool AutoAcknowledge { get; set; }
-
-        [XmlAttribute("compressed")]
-        public bool Compressed { get; set; }
-
+        public String Name { get; set; }
+        public String QueueUrl { get; set; }
+        public Int32 MaxMessages { get; set; }
+        public Boolean AutoAcknowledge { get; set; }
+        public Boolean Compressed { get; set; }
```

Comment:
Any reason for using `String`, `Int32`, and `Boolean` here instead of `string`, `int`, and `bool`?